scheduler/src/cook/mesos/api.clj

```diff
@@ -1059,7 +1064,7 @@
   to Datomic.
   Preconditions:  The context must already have been populated with both
   ::jobs and ::groups, which specify the jobs and job groups."
-  [conn {:keys [::groups ::jobs] :as ctx}]
+  [conn {:keys [::groups ::jobs] :as ctx} force-container-user?]
```

Comment:
`force-container-user?` is declared as the *third* argument to `create-jobs!` here. I think it should be the *second* argument; i.e., it should be moved right after `conn`. The context needs to be the last argument because all of the other arguments are passed via a partial function application, and then the context is passed implicitly by the Liberator framework. (See my other comment below for the partial function application.)